### PR TITLE
Output build id as msbuild property in PushMetadataToBuildAssetRegistry

### PR DESCRIPTION
**Pull request by zivkan**
fixes https://github.com/dotnet/arcade/issues/5122

Please note that I'm not an MSBuild expert. I don't know if creating this output parameter will have consequences on targets that call the task and don't handle the output.

### CODE REVIEW

File: src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs [hunk 1]
```diff
@@ -39,6 +39,9 @@ public class PushMetadataToBuildAssetRegistry : MSBuild.Task, ICancelableTask
 
         public string RepoRoot { get; set; }
 
+        [Output]
+        public string BuildId { get; set; }
```
Comment (JohnTortugo) [inline src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs]: You can make it an _int_ I suppose?

Comment (zivkan) [reply src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs]: msbuild supports int? 😛

**Comment by riarenas** [comment]
Change looks reasonable to me.

**Comment by JohnTortugo** [comment]
@zivkan - After this is merged and assets pushed to some feed we'll still need to update the Arcade.SDK to make use of the newer version of the task. It's not automatic, unfortunately.

**Pull request merged**